using-mobile-extensions/mobile-core/rules-engine/rules-engine-details.md

```diff
@@ -373,25 +368,20 @@ If the conditions pass, the in-app message is displayed \(first consequence\), a
 }
 ```
 
-## Rules URLs
+## Rules URL
 
-Since the Rules Engine can support multiple end-point URLs from where to fetch rules, the URLs are prioritized based on the following scheme:
+Rules Engine requires an endpoint URL to be configured in the remote configuration, specifying from where to fetch rules:
```

Comment:
change it to `be configured in the configuration`, since it can also be set in the code.